### PR TITLE
Better error when failing to save roles

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?221272

easy :fish: @snopoke / anyone

### CODE REVIEW

File: corehq/apps/users/views/__init__.py [hunk 1]
```diff
@@ -550,6 +551,7 @@ def undo_remove_web_user(request, domain, record_id):
 # If any permission less than domain admin were allowed here, having that permission would give you the permission
 # to change the permissions of your own role such that you could do anything, and would thus be equivalent to having
 # domain admin permissions.
+@json_error
```
Comment (snopoke) [inline corehq/apps/users/views/__init__.py]: didn't know we had this

**Comment by snopoke** [comment]
:+1: need to wait for tests?

**Comment by czue** [comment]
couldn't hurt to wait

**Comment by czue** [comment]
:white_check_mark:

**Pull request merged**